steward/src/commands/schedule_corks/aave_cellar/shutdown.rs

```diff
@@ -0,0 +1,78 @@
+use crate::{
+    application::APP,
+    cellars::{self},
+    config,
+    prelude::*,
+    somm_send,
+};
+use abscissa_core::{clap::Parser, Application, Command, Runnable};
+use deep_space::{Coin, Contact};
+use ethers::abi::AbiEncode;
+use somm_proto::cork::Cork;
+use std::time::Duration;
+use steward_abi::aave_v2_stablecoin::*;
+const MESSAGE_TIMEOUT: Duration = Duration::from_secs(10);
+const CHAIN_PREFIX: &str = "somm";
+
+/// Shutdown subcommand
+#[derive(Command, Debug, Parser)]
+#[clap(
+    long_about = "DESCRIPTION \n\n Shutdown target Cellar.\n This command schedules the shutdown of a Cellar. This is a validator only \n command and can only be run by validators. It Schedules shutdown based on the height specified by \n the validators. Therefore, it'll execute the function when the chain reaches that height."
+)]
+pub struct ShutdownCmd {
+    /// Target contract for scheduled cork.
+    #[clap(short, long)]
+    contract: String,
+
+    /// Block height to schedule cork.
+    #[clap(short = 'b', long)]
```

Comment:
Probably want `h` as the short code here for `height`.

Reply:
The reason the `block_height`'s shortcode is `b` is because `h` is taken for `help` in all of Abscissa's commands. If I go-ahead to add `h` for `block_height`, I'll get this error:

```
thread 'main' panicked at 'App shutdown: Short option names must be unique for each argument, but '-h' is in use by both 'height' and 'help'', /home/ugochi/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-3.0.12/src/build/app/debug_asserts.rs:100:17
```

Reply:
👍